### PR TITLE
Multi loss

### PR DESCRIPTION
**Pull request by lhendre**
WIP, providing two additional loss functions. Custom is a WIP but there is a built in cross entropy one in pytorch

### CODE REVIEW

**Comment by hannahg141** [comment]
consolidated into [this PR](https://github.com/hannahg141/ClimateNet/pull/1/files)
